src/test/groovy/org/ods/e2e/openshift/OpenShiftSpec.groovy

```diff
@@ -0,0 +1,57 @@
+package org.ods.e2e.openshift
+
+import geb.spock.GebReportingSpec
+import org.ods.e2e.openshift.pages.ConsoleCatalogPage
+import org.ods.e2e.openshift.pages.ConsoleProjectsPage
+import org.ods.e2e.openshift.pages.OpenShiftLoginPage
+import org.ods.e2e.openshift.pages.OpenShiftLoginSelectorPage
+import org.ods.e2e.util.BaseSpec
+import org.ods.e2e.util.SpecHelper
+import spock.lang.Ignore
+
+class OpenShiftSpec extends BaseSpec {
+    static Properties applicationProperties = new SpecHelper().getApplicationProperties()
+    String projectName
+
+    def setup() {
+        baseUrl = applicationProperties."config.openshift.url"
+        projectName = applicationProperties."config.project.key"
+    }
+
+    def "can login to OpenShift"() {
+        when: "Visit openshift login page"
+        doOpenshiftLoginProcess()
+
+        then: "We are in the console page"
+        at ConsoleCatalogPage
+        report()
+    }
+
+    @Ignore
```

Comment:
@jorge-romero @Ignore'd tests should not be part of the code. Can we fix this?

Reply:
I'll remove all the ones that I have as tagged as ignored that were useful for initial tests